docs/user-guide/helm.md

```diff
@@ -498,3 +498,24 @@ spec:
     helm:
       skipCrds: true
 ```
+
+
+## Helm `--skip-tests`
+
+By default, Helm includes test manifests when rendering templates.
+Argo CD will assume that skipTests is true unless it is explicitly set to false within the Argo CD Application.
```

Comment:
If I understand correctly, `--skip-tests` is `false` by default in Helm. So changing the default in Argo CD to `true` would be a breaking change. If I'm correct, then I think we should switch the default to `false`, even if it's a somewhat less-nice UX.

Reply:
I've made the changes to adjust the default setting of --skip-tests to false in Argo CD. Please take a look when you have time and let me know if any further adjustments are needed.